app/Http/Controllers/DocuSignController.php

```diff
@@ -254,6 +304,28 @@ public function handleProviderCallback(Request $request)
         }
 
         switch ($request->session()->get('next')) {
+            case 'getUserToken':
+                // In this case, a user has just authenticated with DocuSign from the Nova menu option.
+                // We need to store the credentials in their user model, then send them back to Nova.
+
+                /** @var \App\Models\User $user */
+                $user = $request->user();
+
+                if ($userinfo->getEmail() !== $user->uid.'@gatech.edu') {
+                    $userinfo_serialized = DocuSign::serializeUserInfo($userinfo);
+
+                    Log::info($userinfo_serialized);
+
+                    abort(401);
```

Comment:
aborting with a plain 401 should probably at least log a message saying what happened, rather than the user info with no context